src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -256,7 +262,7 @@ private string LoadContentFromResources(string name)
                 {
                     assemblyName = assemblyName.Substring(0, versionIndex);
                 }
-                assembly = AppDomain.CurrentDomain.GetAssemblies().Where(a => a.GetName().Name == assemblyName).FirstOrDefault();
+                assembly = AppDomain.CurrentDomain.GetAssemblies().FirstOrDefault(a => a.GetName().Name == assemblyName);
```

Comment:
This is good point here from the report, in certain case I can see it saves time